src/core/constants.ts

```diff
@@ -17,10 +17,11 @@
 
 import { AccountId, FileId } from '@hashgraph/sdk'
 import { color, type ListrLogger, PRESET_TIMER } from 'listr2'
-import path, { dirname, normalize } from 'path'
-import { fileURLToPath } from 'url'
+import path, { normalize } from 'path'
+import os from 'node:os'
 
-export const ROOT_DIR = path.join(dirname(fileURLToPath(import.meta.url)), '..', '..')
+export const ROOT_DIR = process.cwd()
```

Comment:
I got root_dir to work correctly from any directory testing with `npm install -g @hashgraph/solo`, then running `solo --help`.  

I believe `process.cwd()` will reverse a lot of my fixes.